lib/chef/platform/query_helpers.rb

```diff
@@ -36,6 +36,28 @@ def windows_server_2003?
         is_server_2003
       end
 
+      def windows_nano_server?
+        return false unless windows?
+        require 'win32/registry'
+
+        # This method may be called before ohai runs (e.g., it may be used to
+        # determine settings in config.rb). Chef::Win32::Registry.new uses
+        # node attributes to verify the machine architecture which aren't
+        # accessible before ohai runs.
+        nano = nil
+        key = "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Server\\ServerLevels"
```

Comment:
Wont this be true on all server versions of windows?  From my server 2012 workstation:

```
get-itemproperty "hklm:\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Server\\ServerLevels"


ServerCoreExtended : 1
ServerCore         : 1
Server-Gui-Mgmt    : 1
Server-Gui-Shell   : 1
PSPath             : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server\ServerLevels
PSParentPath       : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server
PSChildName        : ServerLevels
PSDrive            : HKLM
PSProvider         : Microsoft.PowerShell.Core\Registry
```